Makefile

```diff
@@ -90,9 +90,16 @@ mock: ##@other Regenerate mocks
 	mockgen -source=geth/common/types.go -destination=geth/common/types_mock.go -package=common
 	mockgen -source=geth/common/notification.go -destination=geth/common/notification_mock.go -package=common -imports fcm=github.com/NaySoftware/go-fcm
 	mockgen -source=geth/notification/fcm/client.go -destination=geth/notification/fcm/client_mock.go -package=fcm -imports fcm=github.com/NaySoftware/go-fcm
+	mockgen -source=geth/account/interfaces.go -destination=geth/account/types_mock_test.go -package=account
+	mockgen -destination=geth/account/wallet_mock_test.go -package=account github.com/ethereum/go-ethereum/accounts Wallet
 
 test: test-unit-coverage ##@tests Run basic, short tests during development
 
+test-account:
+	build/env.sh go test -v -coverprofile=coverage.out ./geth/account
+	@build/env.sh go tool cover -html=coverage.out -o coverage.html
```

Comment:
Why the use of `@build` here, lets keep consistent with previous `build`?

Reply:
@influx6 `@` means `don't echo the command being run to the stdout`

Reply:
Ah! I see. Got it!

Reply:
I think we don't need it. I've got it with https://github.com/status-im/status-go/pull/409.
I'll delete it.